tests/lean/caching_user_attribute.lean.expected.out

```diff
@@ -3,10 +3,12 @@
 eq.refl
 eq.mp
 
+[user_attributes_cache] no cached result for [foo]
+[user_attributes_cache] recomputing cache for [foo]
 eq.refl
 eq.mp
 
-[user_attributes_cache] cached result for [foo] has been found, but cache fingerprint does not match
+[user_attributes_cache] no cached result for [foo]
```

Comment:
This test is probably flaky now after the merge of #1489 (each of the `run_cmd` commands is executed as a separate task, and potentially on different threads).